content/06.know-your-problem.md

```diff
@@ -1,20 +1,28 @@
 ## Tip 4: Know your data and your question {#know-your-problem}
 
-In the era of easily accessible datasets, one sometimes starts analyzing data without a good understanding of the study design, namely why the data were collected and how. 
-Having appropriate meta-data, a comprehensive data dictionary, and even the actual data collection protocol is essential for any analysis, including one that involves deep learning. 
-It's even better to also have access to a subject matter expert who has either collected or analyzed this type of data before! 
-For example, if the main reason why the data were collected was to test the impact of an intervention, it may be the case that a randomized controlled trial was performed.
-However, ethical or other study considerations may make this impossible or impractical, in which case the design may have been an observational one, either prospective or retrospective.
-These designs may also incorporate some amount of matching - for example, cases and controls may be selected so that the age range or weight distribution is similar.
-All of these different designs have different assumptions and caveats, which cannot be ignored during a data analysis.
-Many datasets are now passively collected or do not have a specific design, but even in this case it is important to know how individuals or samples were treated (for example, if all samples are from the same study site, if certain ethnic groups or zip codes are oversampled, if there are differences in processing dates or techniques.)
+Having a well-defined question, a hypothesis to test based on that question, and a clear analysis plan for proving the hypothesis are all crucial for carrying out a successful deep learning project.
+Just like it would be inadvisable to step foot in a laboratory and begin experiments without having a defined endpoint, a deep learning project should not be undertaken without preparation.
+Foremost, it is important to assess if a dataset exists that can prove or disprove the hypothesis, and obtaining said data and associated meta-data should be pursued as early on in the project as possible.
```

Comment:
I meant it in the context of the scientific method and not hypothesis space/null hypothesis testing. I'm okay with having it as the original "good question" wording to avoid confusion.